gui/packages/ubuntupro/integration_test/ubuntu_pro_for_wsl_test.dart

```diff
@@ -199,7 +199,7 @@ void main() {
         expect(tester.widget<FilledButton>(continueButton).enabled, isFalse);
 
         // check that valid input enabled continue, and continue
-        await tester.enterText(fqdnInput, 'example.com');
+        await tester.enterText(fqdnInput, 'localhost');
```

Comment:
Well it consistently failed before, and it no longer fails now.

I ran a very short program:
```
func main() {
	log.Println("Starting")
	_, err := net.Dial("tcp", "example.com:4653")
	log.Fatal(err)
}
```

With example.com it waits for 20s before timing out:
```
dial tcp 93.184.216.34:4653: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
```

Whereas localhost is instantly refused :
```
dial tcp [::1]:4653: connectex: No connection could be made because the target machine actively refused it.
```

Reply:
So the issue is that we dial Landscape server before completing the gRPC call and having it set to a non-existing host is the worse scenario because it must wait the dialing to timeout, right?

Reply:
Looks like it. Maybe a TODO for the future.

Reply:
Note that it is made worse by the fact that `example.com` is a real website. `exadfdfdfzgample.com` fails instantly with:
```
dial tcp: lookup exadfdfdfzgample.com: no such host
```